docs/admin-manual/data-admin/backup-restore/create-repository.md

```diff
@@ -0,0 +1,167 @@
+WITH S3
+ON LOCATION "s3://bucket_name/s3_repo"
+PROPERTIES
+(
+    "s3.endpoint" = "s3.us-east-1.amazonaws.com",
+    "s3.region" = "us-east-1",
+    "s3.access_key" = "ak",
+    "s3.secret_key" = "sk"
+);
+```
+
+- Replace bucket_name with the name of your S3 bucket.
+- Provide the appropriate endpoint, access key, secret key, and region for your S3 setup.
+
+## Creating a Repository for Azure
+
+To create a repository for Azure storage, use the following SQL command:
+
+```sql
+CREATE REPOSITORY `azure_repo`
+WITH S3
+ON LOCATION "s3://bucket_name/azure_repo"
+PROPERTIES
+(
+    "s3.endpoint" = "selectdbcloudtestwestus3.blob.core.windows.net",
+    "s3.region" = "dummy_region",
+    "s3.access_key" = "ak",
+    "s3.secret_key" = "sk",
+    "provider" = "AZURE"
+);
+```
+
+- Replace bucket_name and container with your Azure container information.
+- Provide your Azure storage account and key for authentication.
+
+## Creating a Repository for GCP
+
+To create a repository for Google Cloud Platform (GCP) storage, use the following SQL command:
+
+```sql
+CREATE REPOSITORY `gcp_repo`
+WITH S3
+ON LOCATION "s3://bucket_name/backup/gcp_repo"
+PROPERTIES
+(
+    "s3.endpoint" = "storage.googleapis.com",
+    "s3.region" = "US-WEST2",
+    "s3.access_key" = "ak",
+    "s3.secret_key" = "sk"
+);
+```
+
+- Replace bucket_name with the name of your GCP bucket.
+- Provide your GCP endpoint, region, access key, and secret key.
+
+## Creating a Repository for OSS (Alibaba Cloud Object Storage Service)
+
+To create a repository for OSS, use the following SQL command:
+
+```sql
+CREATE REPOSITORY `oss_repo`
+WITH S3
+ON LOCATION "s3://bucket_name/oss_repo"
+PROPERTIES
+(
+    "s3.endpoint" = "oss.aliyuncs.com",
+    "s3.region" = "cn-hangzhou",
+    "s3.access_key" = "ak",
+    "s3.secret_key" = "sk"
+);
+```
+- Replace bucket_name with the name of your OSS bucket.
+- Provide your OSS access key, secret key, and endpoint.
+
+## Creating a Repository for MinIO
+
+To create a repository for MinIO storage, use the following SQL command:
+
+```sql
+CREATE REPOSITORY `minio_repo`
+WITH S3
+ON LOCATION "s3://bucket_name/minio_repo"
+PROPERTIES
+(
+    "s3.endpoint" = "yourminio.com",
+    "s3.region" = "dummy-region",
+    "s3.access_key" = "ak",
+    "s3.secret_key" = "sk",
+    "use_path_style" = "true"
+);
+```
+
+- Replace bucket_name with the name of your MinIO bucket.
+- Provide your MinIO access key, secret key, and endpoint.
+
+## Creating a Repository for HDFS
+
+```sql
+CREATE REPOSITORY `hdfs_repo`
+WITH hdfs
+ON LOCATION "/prefix_path/hdfs_repo"
+PROPERTIES
+(
+    "fs.defaultFS" = "hdfs://127.0.0.1:9000",
+    "hadoop.username" = "doris-test"
+)
+```
+
+- Replace prefix_path with the real path.
+- Provide your hdfs endpoint and username.
+
+For more detailed usage instructions and examples, refer to the CREATE REPOSITORY documentation (../../sql-manual/sql-statements/data-modification/backup-and-restore/CREATE-REPOSITORY).
```

Comment:
invalid link, please add `[]` for anchor text , such as : **[**CREATE REPOSITORY documentation**]**(../../sql-manual/sql-statements/data-modification/backup-and-restore/CREATE-REPOSITORY).